src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * Implements User interface
+ */
+public class Ui {
+    private Storage store;
+    private TaskList tasks;
+    private Scanner reader;
```

Comment:
i feel generally a camelCase of the class name would be more intuitive than coming up with another name for the class itself. 
```
Storage storage
TaskList taskList
Scanner scanner
```